site/app/templates/officeHoursQueue/QueueStatus.twig

```diff
@@ -190,8 +204,29 @@
   </div>
 {% endif %}
 <script>
-    function toggleSwitchQueueForm(){
+
+    function queueMessage(message, course) {
+        if (Notification.permission === "granted") {
+            // If it's okay let's create a notification
+            let notification = new Notification("SUBMITTY(" + course + ") - New Message From Office Hours Queue Staff: " + message);
```

Comment:
If you look at [https://developer.mozilla.org/en-US/docs/Web/API/Notification/Notification](https://developer.mozilla.org/en-US/docs/Web/API/Notification/Notification) you can see there is another constructor parameter. You can pass a body to the notification rather than using the title to hold the message. There is also some other fields that could let you customize this nicely.